binderhub/build.py

```diff
@@ -320,10 +325,11 @@ def submit(self):
             w = watch.Watch()
             try:
                 for f in w.stream(
-                        self.api.list_namespaced_pod,
-                        self.namespace,
-                        label_selector="name={}".format(self.name),
-                        timeout_seconds=30,
+                    self.api.list_namespaced_pod,
+                    self.namespace,
+                    label_selector="name={}".format(self.name),
+                    timeout_seconds=30,
+                    _request_timeout=(3, 30),
```

Comment:
There is a hardcoded reference here to the same timeout tuple config, and below to (3, None), should perhaps at least this be the same as the constant in utils?

Reply:
yup, I missed this one after I added the constant. Fixed!